pkg/chunk/aws/dynamodb_index_reader.go

```diff
@@ -0,0 +1,238 @@
+package aws
```

Comment:
Given this is used only by the blocksconvert, why not keeping it in its package?

Reply:
It uses several things which are not exported by `aws`